sdk/tables/Azure.Data.Tables/samples/README.md

```diff
@@ -0,0 +1,28 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
```

Comment:
this list comes from the values in [here](https://review.docs.microsoft.com/en-us/new-hope/information-architecture/metadata/taxonomies?branch=master)
I don't see `azure-tables` on it. I am asking on our Teams Channel to see what the proper product name should be

Reply:
looks like:
- azure
- azure-storage
- azure-table-storage